### DISCUSSION TITLE
How to modify the background color of "table-zebra" class?

### DISCUSSION THREAD

**Discussion by iamyourdre** [saadeghi/daisyui#2895] [Q&A] [answered]
I want to change the background color of "table-zebra" class. Is this possible to do?
![image](https://github.com/saadeghi/daisyui/assets/112853804/3e326ca5-81bc-45c9-bd6c-17b313d39f2c)

Oh btw, i have tried using:
`<table className="table odd:bg-p-light even:bg-s-light">`
But i see no difference.

![image](https://github.com/saadeghi/daisyui/assets/112853804/86a320f8-46c0-49df-882b-71458df780ef)

**Comment by Vijaykv5** [comment, marked as answer]
Hi @iamyourdre 

I think Daisy UI does not provide a built-in utility to directly change the background color of the "table-zebra" class.
If you need to customize the background color of the zebra-striped table, you'll likely need to override the existing styles using custom CSS
You can add it in the CSS file like this : 
`.table-zebra tbody tr:nth-child(odd) {
  background-color: #00000
}`

**Reply by iamyourdre** [reply]
It's work fine, thank you @Vijaykv5 

![image](https://github.com/saadeghi/daisyui/assets/112853804/e1f1e32f-fad1-41cc-9533-12027c740c2c)

**Reply by renannprado** [reply]
@Vijaykv5 in daisyui 5, I've tried the same and it didn't work.

Then I tried the below, and it works, but is it the correct/best way to do it?

```css
@layer components {
    .table tbody tr:nth-child(2n) {
        background-color: var(--color-accent) !important;
    }
}
```
